prusa/link/web/wizard.py

```diff
@@ -116,13 +115,17 @@ def process_network(config):
 def process_connect(config):
     """Process Connect section"""
     connect = config[CONNECT]
+    app.wizard.connect_hostname = 'connect.prusa3d.com'
+    app.wizard.connect_tls = 1
+    app.wizard.connect_port = 0
+
     for option in config.options(CONNECT):
-        if option == 'hostname':
-            app.wizard.connect_hostname = connect['hostname']
-        if option == 'tls':
-            app.wizard.connect_tls = connect.getboolean('tls')
-        if option == 'port':
-            app.wizard.connect_port = connect['port']
+        # if option == 'hostname':
```

Comment:
What if i have another connect settings in `prusa_printer_settings.ini` which i want to restore. That is ok, to use my settings.

Reply:
Mistake, I'll fix it